utils/defaulter.go

```diff
@@ -181,7 +191,16 @@ func (d *Defaulter) MustSet(arg interface{}) {
 
 func (d *Defaulter) getEntitySchema(entityType string) (map[string]interface{}, error) {
 	var schema map[string]interface{}
-	schema, err := d.client.Schemas.Get(d.ctx, entityType)
+	endpoint := fmt.Sprintf("/schemas/%s", entityType)
+	if d.isKonnect {
+		entityType = kongToKonnectEntitiesMap[entityType]
+		endpoint = fmt.Sprintf("/v1/schemas/json/%s", entityType)
+	}
+	req, err := d.client.NewRequest(http.MethodGet, endpoint, nil, nil)
+	if err != nil {
+		return schema, err
+	}
+	_, err = d.client.Do(d.ctx, req, &schema)
 	if err != nil {
 		return schema, err
 	}
```

Comment:
we could remove this if and its following line and just do `return schema, err`

Reply:
Totally, good catch, thanks!